src/addons/editors/__tests__/CheckboxEditor.spec.js

```diff
@@ -5,23 +5,44 @@ var TestUtils      = require('react/lib/ReactTestUtils');
 var CheckboxEditor = require('../CheckboxEditor');
 
 describe('CheckboxEditor', () => {
-  var component;
+ var component;
+ var testColumn = {
+   onRowSelect : function(){}
+ };
 
-  describe('Basic tests', () => {
+ describe('Basic tests', () => {
+   beforeEach(() => {
+     spyOn(testColumn, 'onRowSelect');
+     component = TestUtils.renderIntoDocument(<CheckboxEditor
+       value={true}
+       rowIdx={1}
+       column={testColumn}/>);
+     });
+
+     it('should create a new CheckboxEditor instance', () => {
+       expect(component).toBeDefined();
+     });
+
+     it('should be selected if value prop is true', () => {
+       var Input = TestUtils.findRenderedDOMComponentWithTag(component, 'input');
+       expect(Input.props.checked).toBe(true);
+     });
```

Comment:
we should be looking at what we're rending here instead of its props

so 
`
var checkboxNode = Input.getDOMNode();
expect(checkBoxNode.checked).toBe(true)
`

syntax might be slightly wrong